tests/unittest/test_throttler.py

```diff
@@ -90,6 +189,16 @@ def test_hangid(self):
         throttler = Throttler(ConfigManager.from_dict({}))
         assert throttler.throttle(raw_crash) == (REJECT, 'has_hangid_and_browser', None)
 
+    def test_infobar(self):
+        raw_crash = {
+            'ProductName': 'Firefox',
+            'SubmittedFromInfobar': 'true',
+            'Version': '52.0.2',
```

Comment:
glad to see a more complex version number than major + minor

Reply:
I can sprinkle in a few more.